src/modules/organisations/organisations.controller.ts

```diff
@@ -88,6 +92,22 @@ export class OrganisationsController {
     return this.organisationsService.getOrganisationMembers(org_id, page, page_size, sub);
   }
 
+  @UseGuards(MembershipGuard)
+  @Get(':org_id/members/search')
+  @ApiOperation({ summary: 'Search for members within an organization' })
+  @ApiResponse({ status: 200, description: 'User(s) found successfully', type: SearchMemberResponseDto })
+  @ApiResponse({ status: 403, description: 'User does not have permission' })
+  @ApiResponse({ status: 404, description: 'Organisation Not Found' })
+  @ApiResponse({ status: 500, description: 'An error occured internally' })
```

Comment:
Internal error isn't what you want your client to see in production. All endpoint is to expect this by default when the error isn't from the client. Please remove